cmake/common.cmake

```diff
@@ -595,14 +595,16 @@ function(add_e2e_test)
   "ADDITIONAL_ARGS"
   )
 
-  add_test(
-    NAME ${PARSED_ARGS_NAME}
-    COMMAND ${PYTHON} ${PARSED_ARGS_PYTHON_SCRIPT}
-      -b .
-      --label ${PARSED_ARGS_NAME}
-      ${CCF_NETWORK_TEST_ARGS}
-      ${PARSED_ARGS_ADDITIONAL_ARGS}
-  )
+if (BUILD_END_TO_END_TESTS)
```

Comment:
Minor: Please indent this to match the `endif()`.

Reply:
Done, sorry.